ui.apps/src/main/content/jcr_root/apps/etoolbox-query-kit/components/console/queryStatistic/queryStatistic.html

```diff
@@ -0,0 +1,95 @@
+<sly data-sly-use.queryStatisticModel="com.exadel.etoolbox.querykit.core.models.query.QueryStatisticModel"></sly>
+<div class="query-statistic-headlines">
+    <a id="by-usage"><h1>Indexes by usage</h1></a>
+    <span class="nav"><a href="#all">All queries</a></span>
+    <span class="nav"><a href="#by-index">Queries by index</a></span>
+</div>
+<br/>
+<sly data-sly-list.index="${queryStatisticModel.indexesByUsage}">
+    <a data-sly-test="${queryStatisticModel.indexesByUsage[index] != 0}" href="#${index}" class="quantity">
+        ${index}
+    </a>
+    <sly data-sly-test="${queryStatisticModel.indexesByUsage[index] == 0}">${index}</sly>
+    ${queryStatisticModel.indexesByUsage[index]}&nbsp;&nbsp;&nbsp;
+</sly>
+
+<!--/*<ul data-sly-list.index="${queryStatisticModel.indexesByUsage}">*/-->
```

Comment:
Do we need this part?)

Reply:
no, it was just for demo. Deleted unused code